src/main/java/uk/ac/cam/cl/dtg/isaac/dos/PgLocationHistory.java

```diff
@@ -1,350 +0,0 @@
-/**
- * Copyright 2015 Stephen Cummins
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- *
- * You may obtain a copy of the License at
- * 		http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package uk.ac.cam.cl.dtg.isaac.dos;
-
-import java.io.IOException;
-import java.sql.Connection;
-import java.sql.PreparedStatement;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-import java.sql.Statement;
-import java.util.Collection;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-
-import org.apache.commons.lang3.Validate;
-import org.postgresql.util.PGobject;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import uk.ac.cam.cl.dtg.segue.dao.SegueDatabaseException;
-import uk.ac.cam.cl.dtg.segue.database.PostgresSqlDb;
-import uk.ac.cam.cl.dtg.util.locations.Location;
-import uk.ac.cam.cl.dtg.util.locations.PostCode;
-
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.google.api.client.util.Lists;
-import com.google.api.client.util.Maps;
-import com.google.inject.Inject;
-
-/**
- * @author sac92
- *
- */
-public class PgLocationHistory implements LocationHistory {
-    private final PostgresSqlDb database;
-
-    private static final Logger log = LoggerFactory.getLogger(PgLocationHistory.class);
-    
-    /**
-     * PgLocationHistory.
-     * 
-     * @param database
-     *            - Preconfigured PostGres instance.
-     */
-    @Inject
-    public PgLocationHistory(final PostgresSqlDb database) {
-        this.database = database;
-    }
-
-    /*
-     * (non-Javadoc)
-     * 
-     * @see uk.ac.cam.cl.dtg.isaac.dos.LocationHistory#getLatestByIPAddress(java. lang.String)
-     */
-    @Override
-    public LocationHistoryEvent getLatestByIPAddress(final String ipAddress) throws SegueDatabaseException {
-        Validate.notBlank(ipAddress);
-
-        String query = "SELECT * FROM ip_location_history WHERE ip_address = ? AND is_current = ? ORDER BY last_lookup DESC";
```

Comment:
I think this table could now be removed, but could also understand the value in keeping existing data for historical purposes.